roles/kubernetes/kubeadm/tasks/main.yml

```diff
@@ -57,6 +57,8 @@
     --ignore-preflight-errors=all
   register: kubeadm_join
   when: not is_kube_master and (not kubelet_conf.stat.exists)
+  environment:
+    PATH: "{{ bin_dir }}:{{ ansible_env.PATH }}"
```

Comment:
Seems wrong that this should be the only task that is missing `PATH`

Reply:
this is really strange, maybe it's not centos problem, but some problem with env PATH settings in some OS.

Reply:
but anyway it will not make any other harm, so approving